DEV-2459: Review and Improvements (#643)
cloudposse/docs: docs/layers/accounts/prepare-aws-organization.mdx

import KeyPoints from '@site/src/components/KeyPoints';
import Steps from '@site/src/components/Steps';

> The Cold Start is the most manually involved tier. Read through the following steps and see [the detailed documentation](/layers/accounts/tutorials/manual-configuration/) for edge cases.

Reviewer: ```suggestion
The Cold Start involves more manual steps than layers. Read through the following steps and see [the detailed documentation](/layers/accounts/tutorials/manual-configuration/) for edge cases.
```

import KeyPoints from '@site/src/components/KeyPoints';
import Steps from '@site/src/components/Steps';

> The Cold Start involves more manual steps than layers. Read through the following steps and see [the detailed documentation](/layers/accounts/tutorials/manual-configuration/) for edge cases.

Reviewer: ```suggestion
The Cold Start involves more manual steps than other layers. Read through the following steps and see [the detailed documentation](/layers/accounts/tutorials/manual-configuration/) for edge cases.
```

Author: added

| Vendor components         | `atmos workflow vendor -f baseline` |
| Configure root SuperAdmin | Click Ops                           |

:::tip Cold Start

> The set up process for the "baseline" or "account" tier is commonly referred to as the Cold Start.

Reviewer: Tier or layer?

Author: layer
